package.nls.it.json

```diff
@@ -1,4 +1,5 @@
 {
+  "command.activateIcons.title": "",
```

Comment:
And I suppose something like this here:
`"command.activateIcons.title": "Attiva tema icone VSCode"`